### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Keyboard: MAS2.1.1: Cannot able to navigate among radio buttons using Arrow keys in "Common options" window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#446]
### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to ""Start Profiling"" option and invoke it.
5. This will invokes ""Xamarin Profiler"" application.

### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations" and invoke next button. This will invokes Common options window.
3. Navigate to "Max Frames" edit box and change the value.
4. Press Tab and observe the keyboard focus.
5. Repeat the same with Shift+Tab.
6. Please do the same for "Enable Automatic snapshots" edit box.

### Expected Behavior
Should be able to navigate through the radio buttons using arrows.
### Actual Behavior
Cannot able to navigate among radio buttons using Arrow keys in "Common options" window.
Note: Using Tab , user is able to navigate through radio buttons.
### User Impact
User expects same level radio buttons to be navigated by Arrow Keys.
### Screenshot
![radio buttons](https://user-images.githubusercontent.com/22956130/41720558-0ccd2318-7581-11e8-95f7-a409c5c0c6cb.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS2.1.1
Xamarin-Profiler-MAC
Wipro
